setup.py

```diff
@@ -89,17 +89,10 @@
       author_email='',
       url='',
       keywords='web fedora pyramid',
-      packages=find_packages(),
+      packages=find_packages(exclude=['tests']),
       include_package_data=True,
       zip_safe=False,
       install_requires = requires,
-      tests_require = [
-          'nose',
-          'nose-cov',
-          'webtest',
-          'mock'
-      ],
-      test_suite="nose.collector",
```

Comment:
Hello @jarifibrahim! Thanks for the PR. I think we want to keep the above two elements so that the tests can be run with ```python setup.py test```. It's my fault for not being clear in the issue I filed, but to be more explicit, what I wanted was to make it so ```python setup.py install``` doesn't install the tests. I think what you did on line 92 will accomplish that goal.

Can you add the tests_require and test_suite back? I'm very sorry for not making that clear in my issue, and thanks for the patch!